db/fixtures/miq_event_definitions.csv

```diff
@@ -166,6 +166,11 @@ request_service_stop,Service Stop Request,Default,service_process
 service_stopped,Service Stopped,Default,service_process
 
 #
+# Orchestration
+#
+request_orchestration_stack_retire,Orchestration Stack Retire Request,Default,orchestration
```

Comment:
@lfu This needs a corresponding miq_event_definition_set instance for `orchestration` added to [db/fixtures/miq_event_definition_sets.csv](https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_event_definition_sets.csv).

Following the existing pattern for VM and Service the set should be called `orchestration_process`